packages/python/accern_xyme/accern_xyme.py

```diff
@@ -891,16 +891,26 @@ def get_kafka_error_topic(self) -> str:
         assert res is not None
```

Comment:
write function that reads both error topics and matches them up using the id path (note, the function assumes json)